.github/workflows/main.yml

```diff
@@ -90,6 +90,9 @@ jobs:
 
       - name: Run tests on ${{ matrix.browser }}
         run: npm test -- --browsers ${{ matrix.browser }}
+      
+      - name: Run bundlesize
```

Comment:
Adding this task to the `test` job will run the same thing on every test target. It might be better to put this `run` job, which can run the specified NPM scripts.